src/main/drivers/accgyro_spi_mpu6000.c

```diff
@@ -99,12 +99,10 @@ static bool mpuSpi6000InitDone = false;
 #define MPU6000_REV_D9 0x59
 #define MPU6000_REV_D10 0x5A
 
-#define DISABLE_MPU6000       IOHi(mpuSpi6000CsPin)
-#define ENABLE_MPU6000        IOLo(mpuSpi6000CsPin)
+#define DISABLE_MPU6000       IOHi(spiCsnPin)
```

Comment:
This may be acceptable for testing, but should not end up in production code ...